README.md

```diff
@@ -10,22 +10,59 @@ Jupyter notebooks hosted on `stitches` use functionality that is contained withi
 
 **NOTE:**  Ensure you are using Python >= 3.6.  Calling `python` may use a different instance.  Some users may need to use `python3` or the like instead.
 
-`stitches`  users and developers agree to must agree to our community guidelines outlines in our [contirbutors guidelines](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md). Open an issue to ask for help or report an issue ([how to open a github issue](https://docs.github.com/en/enterprise-server@3.1/issues/tracking-your-work-with-issues/creating-an-issue)). 
+`stitches`  users and developers agree to must agree to our community guidelines outlines in our 
```

Comment:
some extra words:
users and developers must agree to our community guidelines outlines in our